src/frontend/js/components/UserLogin/index.tsx

```diff
@@ -43,6 +44,7 @@ export const UserLogin = ({ loginUrl, logoutUrl, signupUrl }: UserLoginProps) =>
    * - `null` when the user is anonymous or the request failed;
    * - a user object when the user is logged in.
    */
+  const intl = useIntl();
```

Comment:
Hmm, I prefere to keep the `<FormattedMessage />` component because I found the syntax clearer. I would use `intl.formatMessage` only to pass translated string in props.
```js
<a href="#">
  <FormattedMessage {...messages.logIn} />
</a>
```

```js
<a href="#">
  {intl.formatMessage(messages.logIn)}
</a>
```